tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -184,20 +186,31 @@ def test_positive_end_to_end_ad(session, ldap_tear_down, ad_data):
     :CaseLevel: Integration
 
     :CaseImportance: High
+
+    :parametrized: yes
     """
+    if ldap_auth_source['auth_type'] == 'ipa':
+        ldap_server_type = LDAP_SERVER_TYPE['UI']['ipa']
+        attr_login = LDAP_ATTR['login']
+    elif ldap_auth_source['auth_type'] == 'ad':
+        ldap_server_type = LDAP_SERVER_TYPE['UI']['ad']
+        attr_login = LDAP_ATTR['login_ad']
+    else:
+        ldap_server_type = LDAP_SERVER_TYPE['UI']['posix']
+        attr_login = LDAP_ATTR['login']
```

Comment:
Above code is repeatable in the multiple tests, it would be good if can place this in the ldap_auth_source fixture itself. 
e.g.
```python
        if request.param.lower() == 'ipa':
            ldap_data['ldap_server_type'] = LDAP_SERVER_TYPE['UI']['ipa']
```

Reply:
Done